source/src/molecule-unfolding/cdk/utils/custom-resource.ts

```diff
@@ -0,0 +1,68 @@
+import * as cdk from 'aws-cdk-lib';
+import {
+  Stack
+} from 'aws-cdk-lib';
+import {
+  Construct
+} from 'constructs'
+
+export default (scope: Construct) => {
+  const account_id = Stack.of(scope).account
+
+  const crossEventRegionCondition = new cdk.CfnCondition(scope, 'CrossEventRegionCondition', {
+    expression: cdk.Fn.conditionNot(
+      cdk.Fn.conditionEquals(cdk.Stack.of(scope).region, 'us-west-2'),
+    ),
+  });
+
+  const provider = cdk.CustomResourceProvider.getOrCreateProvider(scope, 'Custom::CreateEventRule', {
+    codeDirectory: `${__dirname}/custom-resource-lambda/create-event-rule`,
```

Comment:
use cdk `NodejsFunction` instead to remove the `package.json`, `package-lock.json`

Reply:
Will not change to use NodejsFunction, the code use high level cdk.CustomResourceProvider.getOrCreateProvider is more simple and convenient than used raw lambda, even cdk.NodejsFunction.